rotkehlchen/tests/unit/test_solana.py

```diff
@@ -168,7 +168,7 @@ def test_query_tx_from_rpc(solana_manager: 'SolanaManager') -> None:
 
 @pytest.mark.vcr
 def test_query_signatures_for_address(solana_manager: 'SolanaManager') -> None:
-    signatures = solana_manager.query_tx_signatures_for_address(
```

Comment:
no need to initialise the `SolanaManager` here and above. use the fixture for solana inquirer.